src/corona_hakab_model/analyzers/config.py

```diff
@@ -0,0 +1,5 @@
+from os import path
+from project_structure import OUTPUT_FOLDER
+
+POPULATION_OUTPUT_PATH = path.join(path.dirname(path.abspath(__file__)), "../../../output")
```

Comment:
Why not use OUTPUT_FOLDER? I dont think we want multiple output folders, it is confusing

Reply:
OUTPUT_FOLDER ןis not the correct folder,
we have a open issue to organize tall the outputs from the simulation 2 one place, but right now we do have 2 different output_folders in out repo
so i organized it to one place to allow quick change once the  outputs unified